openlibrary/templates/lib/history.html

```diff
@@ -43,7 +43,34 @@ <h2 class="collapse inline">
                 $if page.key.startswith("/books"):
                     /
                     <a rel="nofollow" href="$opds">OPDS</a>
+                $if show_wikipedia_citation_link:
+                    | <a aria-controls="wikicode" id="wikilink" class="dialog--open" title="Cite this on Wikipedia">Wikipedia citation</a>
```

Comment:
```suggestion
                    | <a aria-controls="wikicode" id="wikilink" class="dialog--open" title="Cite this on Wikipedia" href="javascript:;">Wikipedia citation</a>
```

Reply:
Fyi it seems suggested changes don't show on mobile so I hadn't realised this was actionable!